src/chapters/jdbc-auth.xml

```diff
@@ -413,12 +509,29 @@ postgresql-password: <replaceable>some_password</replaceable></programlisting>
                         <row>
                             <entry><property>mysql-password</property></entry>
                             <entry><property>postgresql-password</property></entry>
+                            <entry><property>sqlserver-password</property></entry>
                             <entry>
                                 <para>The password Guacamole should provide when authenticating with
                                     the database. This is given as "some_password" in the examples
                                     given in this chapter.</para>
                             </entry>
                         </row>
+                        <row>
+                            <entry></entry>
+                            <entry></entry>
+                            <entry><property>sqlserver-driver</property></entry>
```

Comment:
Since this property is unique to SQL Server, it might be best documented separately, rather than in the table of required properties common to all.

Reply:
Moved.